container/container.go

```diff
@@ -188,6 +188,13 @@ func (container *Container) SetupWorkingDirectory() error {
 	if container.Config.WorkingDir == "" {
 		return nil
 	}
+
+	// If can't mount container FS at this point (eg Hyper-V Containers on
+	// Windows) bail out now with no action.
+	if !container.canMountContainerFS() {
```

Comment:
@cpuguy83 Urm - I can't remove `container` on that line, unless I change the func to `func canMountContainerFS(container* container)`. Not sure what you want me to do.